source/connect.txt

```diff
@@ -152,40 +239,42 @@ To rename a connection to a deployment in |vsce|:
 
 .. _vsce-disconnect-task:
 
-Disconnect from a Deployment
+Disconnect from a Connection
```

Comment:
This edit seems a little off to me. Why is are we replacing "Connection" with "Deployment" here?

Reply:
This was to match the verbiage of the other subtasks, but I can agree it sounds a little odd so I will roll it back. @jeff-allen-mongo